### PR TITLE
GraphQL: Replace list of company role permissions with a structural view

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) replaces an unindented list of company role permissions with a table that better shows the hierarchy. I also moved this table/list to the top of the `createCompanyRole` and `updateCompanyRole` mutations.

## Affected DevDocs pages

-  ...

## Links to Magento source code

-  ...

### CODE REVIEW

File: src/guides/v2.4/graphql/mutations/create-company-role.md [hunk 1]
```diff
@@ -6,7 +6,11 @@ contributor_link: https://www.atwix.com/
 b2b_only: true
 ---
 
-Use the `createCompanyRole` mutation to create a new company role and permissions.
+The `createCompanyRole` mutation defines a new company role. To create a role, you must provide an array of permissions that determine which resources the role user can access.
```
Comment (erikmarr) [inline src/guides/v2.4/graphql/mutations/create-company-role.md:9]:
```suggestion
The `createCompanyRole` mutation defines a new company role. To create a role, you must provide an array of permissions that determine which resources the user can access.
```

File: src/guides/v2.4/graphql/mutations/create-company-role.md [hunk 2]
```diff
@@ -6,7 +6,11 @@ contributor_link: https://www.atwix.com/
...
+
+{% include webapi/b2b_roles.md %}
+
+Also, you can get the list of all resources defined within the company using [`company`]({{page.baseurl}}/graphql/queries/company.html) query.
```
Comment (erikmarr) [inline src/guides/v2.4/graphql/mutations/create-company-role.md:13]:
```suggestion
Also, you can get the list of all resources defined within the company using the [`company`]({{page.baseurl}}/graphql/queries/company.html) query.
```

File: src/guides/v2.4/graphql/mutations/update-company-role.md [hunk 3]
```diff
@@ -8,7 +8,9 @@ b2b_only: true
 
 Use the `updateCompanyRole` mutation to update the company role and permissions.
 
-You can get the role ID with the [`company`]({{page.baseurl}}/graphql/queries/company.html) query.
+You can get the role ID and the list of all resources defined within the company using [`company`]({{page.baseurl}}/graphql/queries/company.html) query.
```
Comment (erikmarr) [inline src/guides/v2.4/graphql/mutations/update-company-role.md:11]:
```suggestion
You can get the role ID and the list of all resources defined within the company using the [`company`]({{page.baseurl}}/graphql/queries/company.html) query.
```

**Review by erikmarr — APPROVED** [review]
minor suggestions

**Comment by keharper** [comment]
running tests

**Pull request merged**